### PR TITLE
Add customizable section name feature

### LINKED ISSUE #38: [FEATURE] Customize section on the readme file
### Description

Currently this actions looks for 

`<!--START_SECTION:data-section-->`
and 
`<!--END_SECTION:data-section-->`

The idea is to allow us to set the `data-section` name instead of a fixed one, so we can have multiple data sections in a single file that will be updated by different actions...

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by guibranco**
## Fixes Issue

Closes #38

## Changes proposed

- [x] Allow customization of `section-name` in the `file`.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

- The default value is still present in the actions.yml, so no breaking changes
- This allows having multiple different sections on the same file (which is something that I use in some of my projects)

### CODE REVIEW

**Comment by guibranco** [comment]
@eddiejaoude could you please review this PR? Thanks in advance!

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the PR. What are the new files `tech stack` for?

**Comment by guibranco** [comment]
> Thank you for the PR. What are the new files `tech stack` for?

Hi @eddiejaoude. This is to show the stack used by the repository, which is a feature from [StackShare](https://stackshare.io) website. 

I merged it with my branch. Let me know if you need any other changes.

File: dist/index.js [hunk 1]
```diff
@@ -3690,40 +3691,40 @@ function removeHook (state, name, method) {
...
-
-    // export in node or browser
-    if (true) {
-        if ( true && module.exports) {
-            exports = module.exports = chunk;
-        }
-        exports.chunk = chunk;
-    } else {}
-
-}.call(this));
```
Comment (eddiejaoude) [inline dist/index.js:3726]: this whole section looks the same?

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks good, but one section has changes but I can't see the actually changes, please check and only have changes needed in the PR so we can merge

File: dist/index.js (hunk 1 above)
Comment (guibranco) [reply dist/index.js:3726]: I ever reset the file, and the changes still continue...

**Pull request closed without merging**